src/mmw/apps/geoprocessing_api/tasks.py

```diff
@@ -47,18 +48,21 @@ def start_rwd_job(location, snapping, simplify, data_source):
     rwd_url = 'http://%s:%s/%s/%f/%f' % (RWD_HOST, RWD_PORT, end_point,
                                          lat, lng)
 
-    query_string = '?'
+    params = {}
 
     # The Webserver defaults to enable snapping, uses 1 (true) 0 (false)
     if not snapping:
-        query_string += 'snapping=0&'
+        params['snapping'] = 0
 
     # RWD also defaults to simplify the shape according to a tolerance.
     # Passing it `?simplify=0` returns the unsimplified result.
-    if not simplify:
-        query_string += 'simplify=0'
+    if simplify:
```

Comment:
`0` is falsey, so it will never get added as a param. I think we can just add `simplify` to the param list regardless of value and the RWD service will handle the null?

Reply:
Ooof. I pushed a commit to change this to

```py
if not simplify is False:
```

which may merit some parentheses to indicate the grouping.